marklogic-data-hub/src/main/resources/ml-modules/root/data-hub/5/impl/flow.sjs

```diff
@@ -258,6 +258,7 @@ class Flow {
     this.globalContext.sourceDatabase = combinedOptions.sourceDatabase || this.globalContext.sourceDatabase;
 
     if (!(combinedOptions.noBatchWrite || combinedOptions.disableJobOutput)) {
+      Object.assign(flowStep.options, combinedOptions);
```

Comment:
We need to get a test in place for this first; it's likely best for that to be a JUnit test so that we can test a user providing runtime options. 

@srinathgit  @ryanjdew  I'm also wondering about the fix here. The schema for "Batch.schema.json" describes "step" in the Batch document as having a copy of the step. That is an accurate description. 

I think what 2445 is really saying is "The Batch document doesn't account for the runtime options that I provide". 

I am wondering if the right solution here is to add "combinedOptions" to the Batch document. We could also add just "runtimeOptions" so that the user can see what they provided.

Reply:
Done.

Reply:
@rjrudin 
I think it would be best if @yasht2010 can weigh in and confirm if it's ok to add "combinedOptions" and "runtimeOptions" in addition to "options" from step or if "options" in batch doc is the "combinedOptions" that is used in the step.